### PR TITLE
feat: only request auth endpoints when possible

### LINKED ISSUE #1391: Can't see the menu to at least copy the address when I am a collection colaborator

### LINKED ISSUE #1415: Unpublished collection has wrong tooltip copy
Screen Shot 2021-06-03 at 11 39 04 AM

### PR DESCRIPTION
**Pull request by nicosantangelo**
Depends on https://github.com/decentraland/builder-server/pull/221
Closes #1391 
Closes #1415

### CODE REVIEW

**Comment by nachomazzara** [comment]
Thanks nico! 

I have tested both PRs at the same time.

Comments:

- I can still see the individual items as `/item-editor?collection=94594f75-10cf-481a-99f6-dc5a05cc3c13&item=e6e9fef9-153f-4378-ae09-3994e9b04588&realm=localhost-stub` ( `/collection/:id/items`  and `/items/:id` endpoints)

Screen Shot 2021-06-02 at 13 04 35


- I see the collection `/item-editor?collection=94594f75-10cf-481a-99f6-dc5a05cc3c13` (`/collection/:id` endpoint)
Screen Shot 2021-06-02 at 21 44 11


- Not part of this PR but we need to fix the amount in this message. It should be `2` not `2000`
Screen Shot 2021-06-02 at 21 37 28

**Comment by ksullivan1-chwy** [comment]
Well done gang!

**Comment by nachomazzara** [comment]
Thanks!

Few comments:

- If I am not the owner/manager but a member of the committee I can see this: 
https://user-images.githubusercontent.com/7549152/120932323-ac852600-c6cb-11eb-91e2-57b3b375adb1.mov

It would be great to allow only committee members to see the collections under their view `/curation`
(`http://localhost:3000/collections/:collectionId` with a real collection id tested locally)

That will prevent weird things from the committee member as editing the price (ends up crashing as you saw in the view) and publishing a collection that they have not created! 

- Can still see the item if I'm not allowed (creator, manager, committee member). The endpoint that allows it is `http://localhost:5000/v1/collections/:collectionId/items`

Screen Shot 2021-06-06 at 13 46 26

Even if I go to http://localhost:3000/item-editor?collection=c3a2c9b3-5639-4e66-b8f4-46bc8172cf65&realm=localhost-stub I can see the request to `/collections/:collectionId/items`

- I can see `Post to the forum` when I am a manager, but the endpoint only allows the creator of the collection to post. We should remove that option:

Screen Shot 2021-06-06 at 16 30 20

-

File: src/components/ItemEditorPage/RightPanel/RightPanel.tsx [hunk 1]
```diff
@@ -255,6 +255,14 @@ export default class RightPanel extends React.PureComponent<Props, State> {
             const wearableCategories = item ? getWearableCategories(item.contents) : []
             const overrideCategories = item ? getOverridesCategories(item.contents) : []
 
+            console.log({
```
Comment (nachomazzara) [inline src/components/ItemEditorPage/RightPanel/RightPanel.tsx:258]: Ooops

Comment (nicosantangelo) [reply src/components/ItemEditorPage/RightPanel/RightPanel.tsx:258]: 🤦

**Pull request merged**